### DISCUSSION TITLE
Adding Custom Settings to the Sentiment Analysis

### DISCUSSION THREAD

**Discussion by cwwest12** [deepgram/community#772] [General help] [answered]
Hi there,

We are hoping to use the AI sentiment analysis for our in-house call grading platform.  Right now, we have individual Office Managers grading each of their agents' calls.  The can cause subjectivity in the grading.  We are hoping to utilize the sentiment analysis to reach a baseline grade; however, we do have certain criteria that we would like the program to factor in, before making a decision.  We also have four possible grades for each of the following categories: 

- Make a meaningful connection
- Define the reason for the call
- Solve the customers issue/properly manage expectations
- Summarize the call and clearly define next steps

Out of those categories, a grade can be selected, depending on how well the agent embodied the category goal...

- Mastery
- Effective
- Developing
- Not Heard

Do you think we can set those parameters for the sentiment analysis, so that we can get a non-biased, objective, base-line reading?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment, marked as answer]
@cwwest12  There is definitely some potential here with our Sentiment Analysis feature. Given we'll return a sentiment score as an integer value you possibly could map that score on your end the categories that matter to you and assign those categories a range of values for categorization. 

example:

```
Mastery = 0.81 - 1.00
Effective  0.51 - 0.80
Developing 0.31 - 0.50
Not Heard 0.00 - 0.30

```

If you checkout our [Docs on Sentiment Analysis](https://developers.deepgram.com/docs/sentiment-analysis) you can learn more our I'd suggesting trying the feature out with some audio samples using our [API Playground](https://playground.deepgram.com/?endpoint=listen&smart_format=true&sentiment=true&language=en&model=m).
